src/Helpers/TimeSpanHelpers.cs

```diff
@@ -0,0 +1,141 @@
+﻿namespace AzureMcp.Helpers
+{
+    public static class TimeSpanHelpers
+    {
+        /// <summary>
+        /// Parses an ISO 8601 duration string (e.g., PT1M, PT5M, PT1H, P1D) into a TimeSpan
+        /// </summary>
+        /// <param name="iso8601Duration">The ISO 8601 duration string to parse</param>
+        /// <param name="result">The parsed TimeSpan if successful</param>
+        /// <returns>True if parsing was successful, false otherwise</returns>
+        public static bool TryParseISO8601Duration(string iso8601Duration, out TimeSpan result)
```

Comment:
Is there no existing nuget package that we can leverage that already performs this parsing vs re-inventing here?

Reply:
I have updated to use XMLConvert library (I didn't have to install anything fortunately).